tap_anthem/client.py

```diff
@@ -9,21 +9,14 @@
 from singer_sdk.helpers.jsonpath import extract_jsonpath
 from singer_sdk.streams import RESTStream
 
-
 SCHEMAS_DIR = Path(__file__).parent / Path("./schemas")
 
 
-class anthemStream(RESTStream):
-    """anthem stream class."""
-
-    # TODO: Set the API's base URL here:
-    url_base = "https://api.mysample.com"
+class AnthemStream(RESTStream):
```

Comment:
The Anthem data isn't _technically_ served by a RESTful service, but it does return well-formatted JSON when called. I'm keeping things simple by using `RESTStream` (as opposed to rolling my own, and introducing more surface area to test etc).